yarpcconfig/builder.go

```diff
@@ -111,26 +111,29 @@ func (b *builder) Build() (yarpc.Config, error) {
 		var err error
 
 		var ob transport.Outbounds
+		outboundServiceName := ccname
 		if c.Service != ccname {
 			ob.ServiceName = c.Service
+			outboundServiceName = c.Service
```

Comment:
do those 2 line of change simply mean `outboundServiceName := c.Service` ? Right now if `c.Service == ccname`, `outboundServiceName` is set to `ccname`, which is the same as `c.Service`.  In this case, `kit := outboundServiceName(c.Service` is all you need?

Reply:
FIxed :-)